ranked-choice-voting/generate_ballots.py

```diff
@@ -124,18 +124,17 @@ def generate_ballots_2(num_cands: int, amount_of_noise: int) -> None:
 
     df['num_candidates'] = num_cands
     df['noise'] = amount_of_noise
-    df.to_csv(f'../data/election_{file_date}_{num_cands}cands_{amount_of_noise}noise.csv')
+    df.to_csv(f'../data13/election_{file_date}_{num_cands}cands_{amount_of_noise}noise.csv')
 
 
 def ballots_main(num_elections_to_generate):
     for i in range(num_elections_to_generate):  # change to 25k when ready
 
         # rand # cands btwn 3-8, with 3 being the most likely randomly-generated number:
-        cands = np.random.choice([3, 4, 5, 6, 7, 8], 1, p=[.6, .25, .1, .03, .015, 0.005])[0]
+        cands = np.random.choice([3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19,20,21,22,23,24,25,26,27,28,29,30], 1, p=[.6, .25, .1, .03, .015, 0.005])[0]
```

Comment:
I don't believe this will work with the probabilities as-is, right? I think the # of probabilities has to = the # of elements in the first array, no?